aspnet-core/src/TalentV2.Core/DomainServices/Candidates/CandidateManager.cs

```diff
@@ -286,29 +286,39 @@ public async Task<long> CreateRequestCV(CandidateRequestDto input)
                 return default;
 
             var request = await WorkScope.GetAll<Request>()
-                .Where(q => q.Id == input.RequestId)
-                .Select(s => new
-                {
-                    s.UserType,
-                    s.SubPositionId
-                })
-                .FirstOrDefaultAsync();
+             .Where(q => q.Id == input.RequestId)
+             .Select(s => new
+             {
+                 s.UserType,
+                 s.SubPositionId
+             })
+             .FirstOrDefaultAsync();
+
+            var cvRequestCVs = await WorkScope.GetAll<RequestCV>()
+            .Where(q => q.RequestId == input.PresentrequestId && q.CVId == input.CvId && q.IsDeleted == false)
+            .Select(q => q.Id)
+           .FirstOrDefaultAsync();
 
             var cv = await WorkScope.GetAsync<CV>(input.CvId);
-
             var requestCVStatus = input.RequestCVStatus.HasValue ? input.RequestCVStatus.Value : RequestCVStatus.AddedCV;
             var requestCv = new RequestCV
             {
                 RequestId = input.RequestId,
                 CVId = input.CvId,
                 Status = requestCVStatus
             };
-
             var requestCvId = await WorkScope.InsertAndGetIdAsync(requestCv);
 
             await CurrentUnitOfWork.SaveChangesAsync();
+            var cvCapabilityResul = await WorkScope.GetAll<RequestCVCapabilityResult>()
+                    .Where(q => q.RequestCVId == cvRequestCVs && q.IsDeleted == false)
+                    .Select(q => new CVCapabilityResultDto{
+                        CapabilityId = q.CapabilityId,
```

Comment:
1. Capability (for interviewer) panel:
lấy thêm `Interviewer suggest level`, `Interview time`

2.Application Result (For HR only) panel:
lấy trường `Status`, `Final Level`, `Salary`, `Apply Note`, `Onboard Date`